code/etc/system.xml

```diff
@@ -708,6 +708,16 @@
                             <show_in_store>1</show_in_store>
                             <comment>This feature will allow the indexer to remove disabled products from your Algolia index. This is useful if your products are being disabled without dispatching the catalog_product_save_before event, e.g. a custom solution for importing products.</comment>
                         </remove_disabled_products_from_index>
+                        <index_product_on_category_products_update translate="label comment">
+                            <label>Update product on category products update</label>
+                            <frontend_type>select</frontend_type>
+                            <source_model>adminhtml/system_config_source_yesno</source_model>
+                            <sort_order>70</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
+                            <show_in_store>1</show_in_store>
+                            <comment>This feature enables the extension to automatically update the product index when a product is added to or removed from a category via the "Manage Categories" menu.</comment>
```

Comment:
Maybe something like : "By default ....., Setting it to false will disable that behavior. This is useful for ...." would be better